packages/grafana-runtime/src/utils/i18n.ts

```diff
@@ -0,0 +1,18 @@
+type UseTHook = () => (id: string, defaultMessage: string, values?: Record<string, unknown>) => string;
```

Comment:
i thought we said we'd add the package to the package.json to get the correct types instead of copying/handwriting them 🤔

Reply:
We hand-write this type in our main code anyway 🤷 

https://github.com/grafana/grafana/blob/5d45af111015a7611b3753eb05a3fb6c3406ffbc/public/app/core/internationalization/index.tsx#L79 

I would prefer to keep these copy-pasted to make changes to the api more apparent.

Reply:
ah fair didn't realise 👍